pkg/validation/internal/community.go

```diff
@@ -256,10 +252,13 @@ func validateImageFile(checks CommunityOperatorChecks, deprecatedAPImsg string)
 						}
 
 						if verParsed.GE(semVerOCPV1beta1Unsupported) {
-							checks.errs = append(checks.errs, fmt.Errorf("this bundle is %s. Migrate the API(s) "+
-								"for %s or use the OCP labels for compatible version(s). (e.g. LABEL %s='=v4.8')",
+							checks.errs = append(checks.errs, fmt.Errorf("this bundle is %s. Migrate the API(s) for "+
+								"%s or provide compatible version(s) by using the %s annotation in " +
+								"`metadata/annotations.yaml` to ensure that the index image will be geneared " +
+								"with its label. (e.g. LABEL %s='4.6-4.8')",
 								k8sApiDeprecatedInfo,
 								deprecatedAPImsg,
+								ocpLabelindex,
```

Comment:
is there supposed to be two occurrences of `ocpLabelindex` here?

Reply:
1 - using the %s annotation
2 - LABEL %s='4.6-4.8'